Add Medo to RoQua migration guide (#26)
roqua/documentation: docs/05-medo/05-migratie.md

# Migratie van Medo naar RoQua

> De beheeromgeving van RoQua is het afgelopen jaar flink uitgebreid met onder andere deelnemersbeheer, organisatie-specifieke deelnemersinfovelden, bulkacties en betere rapportages. Daarom brengen we alle projectbeheer samen in één systeem en sluiten we Medo.

Reviewer: ```suggestion
Voor het gebruik van RoQua zonder koppeling met een elektronisch patiëntendossier, zoals voor onderzoeksprojecten met gezonde vrijwilligers of patiënten die bij een externe instelling in zorg zijn, maakten we altijd gebruik van Medo. Medo werd dan gebruikt om deelnemers te beheren, bulk-acties te starten of data te exporteren. De afgelopen twee jaar hebben we dit soort functionaliteit toegevoegd aan de beheeromgeving van RoQua. In de loop van de tijd zijn steeds meer nieuwe onderzoeksprojecten hiervan gebruik gaan maken. We zijn nu op een punt beland waarop het onderhouden van Medo niet langer zinvol is. Daarom brengen we alle projectbeheer samen in één systeem en sluiten we Medo.
```
In mijn ervaring begrijpen veel onderzoekers het verschil tussen RoQua en Medo niet. Daarom iets meer toelichting.

> Voor deelnemers verandert er niets — zij blijven uitnodigingen ontvangen via e-mail en sms, precies zoals voorheen. De migratie heeft alleen betrekking op de beheerkant.

Reviewer: ```suggestion
Net zoals in Medo heeft elk dossier in de RoQua beheeromgeving een knop waarmee RoQua wordt geopend. Als je daarop klikt, kom je in de EPD-interface van RoQua. Vanaf dat punt verandert er niets. Vragenlijsten versturen en uitkomsten inzien werkt hetzelfde. Voor deelnemers verandert er ook niets — zij blijven uitnodigingen ontvangen via e-mail en sms, precies zoals voorheen. De migratie heeft dus alleen betrekking op de beheerkant.
```

Author: Weten ze wel wat een EPD-interface is? Qua omschrijving is het natuurlijk meer "de interface voor binnen een epd". Misschien behandelinterface als term? Maar ja, bij onderzoek is het weer geen behandeling... :/

Reviewer: ```suggestion
Net zoals in Medo heeft elk dossier in de RoQua beheeromgeving een knop waarmee RoQua wordt geopend. Vanaf dat punt verandert er niets. Vragenlijsten versturen en uitkomsten inzien werkt hetzelfde. Voor deelnemers verandert er ook niets — zij blijven uitnodigingen ontvangen via e-mail en sms, precies zoals voorheen. De migratie heeft dus alleen betrekking op de beheerkant.
```
Dan dat zinnetje inderdaad beter weglaten.

## Waar vind ik wat?

Veel onderdelen uit de navigatie van Medo zijn terug te vinden in RoQua, soms onder een andere naam of op een andere plek.

### Projectgegevens → Nachtelijke CSV export

In Medo stond de downloadknop voor de data-export op de pagina **Projectgegevens**.

<Screenshot src="/screenshots/medo-migration/export-medo.png" />

In RoQua vind je deze onder **[Nachtelijke CSV export](../admin/data_exports)**.

### Deelnemers → Ongekoppelde dossiers

Het deelnemersoverzicht is de pagina waar je als gebruiker het meest in werkt. In Medo heette dit **Deelnemers**.

<Screenshot src="/screenshots/medo-migration/deelnemers-medo.png" />

In RoQua heet dit **[Ongekoppelde dossiers](../admin/dossiers)**. "Ongekoppeld" betekent dat er geen extern EPD aan gekoppeld is — voor onderzoeksprojecten is dat altijd het geval, dus al je dossiers staan hier. De term "dossiers" wordt in RoQua breder gebruikt dan alleen voor onderzoeksdeelnemers.

#### Wat is er veranderd?

| Medo | RoQua |
|---|---|
| Deelnemer toevoegen | **Maak dossier** |
| Stats verversen | Niet meer nodig — statistieken zijn altijd actueel |
| Lijst importeren / Lijst exporteren | **Importeren** / **Exporteren** |

> | Bulk aanmelden | **Automatische protocollen in bulk starten** (in het linkermenu) |

Reviewer: ```suggestion
| Bulk aanmelden | **Uitnodigen in bulk** (voor handmatige protocollen) of **Automatische protocollen in bulk starten** (beide in het linkermenu) |
```

Author: had ik niet gedaan omdat handmatige protocollen in medoq helemaal niet kan. maar het is idd een nieuwe extra waar ze nu bij kunnen.

> | Onderzoeksnummer (kolom) | **Patiëntnummer** |

Reviewer: ```suggestion
| Onderzoeksnummer (kolom)<br>Dit nummer is alleen zichtbaar in Medo. Er wordt automatisch een apart dossiernummer aangemaakt om een deelnemer uit Medo aan een dossier in RoQua te koppelen (beginnend met *md-...*) | **Patiëntnummer**<br>Het *patiëntnummer* valt rechtstreeks door in de onderzoeksdata. Bij nieuwe deelnemers kan een eigen patiëntnummer worden opgegeven. Voor gemigreerde projecten worden de bestaande dossiernummers uit RoQua (*md-...*) overgenomen. Het patiëntnummer kan wel handmatig worden aangepast. |
```
Wat mij betreft een zeer belangrijk verschil. Ik vraag me wel af hoeveel informatie we hier willen geven, aangezien het ook weer meer verwarring kan opleveren.

Author: Op zich een goed punt. Misschien beter om een losse paragraaf aan te besteden dan dit in de tabel te stoppen? Ik maak daar wel even een wijziging voor, mag je die nog een keer reviewen.

Reviewer: Ja, is beter idd.

Author: Overigens zie je voor gemigreerde dossiers beide nummers in admin nu:

<img width="579" height="475" alt="image" src="https://github.com/user-attachments/assets/ddcc6f8d-02ae-47c1-ae19-fb42c9efd8d2" />

(En nieuw gemaakte dossiers hebben dan niet de `md-` uiteraard)

Author: De tekst staat er. Ik ga alleen nog even knoeien met de screenshots want die zijn niet handig zo (op de preview site te zien)

Reviewer: Tekst lijkt me prima, zo

#### Navigeren binnen een dossier

In Medo kon je op een rij klikken om naar de detailpagina van een deelnemer te gaan, en van daaruit via **"Open in RoQua"** het dossier in RoQua openen.

In RoQua zijn er twee knoppen per dossier:

- **Open** — opent de dossierweergave (was "Open in RoQua" in Medo)
- **Bewerk** — om gegevens zoals naam en e-mailadres aan te passen (niet zichtbaar in voorbeeldmodus)

### Medewerkers → Admin-gebruikers

> De pagina **Medewerkers** in Medo heet in RoQua **[Admin-gebruikers](../admin/users)**.

Reviewer: Algemene vraag: welke rechten krijgen medewerkers?

Author: Uit de migratiescripts:

```
      # Medo roles (see medo/app/models/role.rb and medo/app/models/ability.rb):
      # - coordinator: project admin — people, protocols, exports, user/role management, audit/import.
      # - collector: operational — same person access as coordinator, no exports, imports, roles, or audit.
      # - researcher: read project + run/download exports only (no person mutations).
      RIGHTS_BY_MEDO_ROLE = {
        "coordinator" => ["all"],
        "collector" => %w[dossiers protocols teams],
        "researcher" => %w[data_exports],
      }.freeze
```

Klopt dat inderdaad?

Reviewer: Ik zou coördinatoren de volgende rechten geven:
- `Admingebruikers`
- `Data Exports`
- `Gegevensverwijderverzoeken`
- `Dossiers omnummeren/samenvoegen`
- `Dossiers`
- `E-mailfouten`
- `EPD-gebruikers`

Eventueel kan je ook rechten geven voor `Templates` en `Protocollen`. Het lastige daaraan vind ik dat je echt wel moet weten wat aanpassingen hierin betekenen binnen lopende onderzoeken. Maar aangezien beheerders zichzelf en anderen toch ook alle andere rechten kunnen geven is dit ook weer een wassen neus.

Dataverzamelaars zou ik alleen het `dossiers` recht geven (en eventueel nog `E-mailfouten`).

Author: Ja misschien moet ik nog even die aanpassing doen dat je alleen rechten mag uitdelen die je zelf ook hebt?

Reviewer: Daar ben ik helemaal voor.

### Instellingen

> De pagina **Instellingen** uit Medo is in RoQua opgesplitst over verschillende pagina's binnen de beheermodule. Voor bestaande projecten die al correct zijn ingericht hoeft hier in de regel niets aan te worden aangepast.

Reviewer: Hier zal niet heel veel gebruik van worden gemaakt, vermoed ik. Het gebeurt nog wel eens dat de projecttitel wordt gewijzigd. Dat is misschien nog een mooie toevoeging voor later.
Is er in de beheeromgeving een optie om e-maildomeinen te specificeren?

Author: Organizatie-naam aanpassen lijkt me idd nog wel een leuke admin-uitbreiding.

Nee, roqua heeft geen emaildomein-lijst. Hebben we nog steeds last van flapdrollen die hotmail-accounts willen aanmaken?

Reviewer: Ik zie weinig domeinen voorbijkomen die niet bij een instelling horen. Hooguit een apart studentmail domein.
